yang/frr-nexthop.yang

```diff
@@ -298,6 +303,33 @@ module frr-nexthop {
     }
   }
 
+  /* Contaner for SRv6 segs SIDs */
+  grouping srv6-segs-stack {
+    description
+      "This grouping specifies an SRv6 segs SIDs stack. The segs
+      SIDs stack is encoded as a list of SID stack entries. The
+      list key is an identifier that indicates the relative
+      ordering of each entry.";
+      container srv6-segs-stack {
+        description
+          "Container for a list of SRv6 segs SIDs stack entries.";
+          list entry {
+            key "id";
+            description
+              "List of SRv6 segs SIDs stack entries.";
+            max-elements 4;
```

Comment:
Why do we need to specify a max-elements? Is it really necessary? Or have you taken this from an equivalent OC or IETF SRV6 yang model?

Reply:
> Why do we need to specify a max-elements? Is it really necessary? Or have you taken this from an equivalent OC or IETF SRV6 yang model?

We specified max-elements=4, because of the next reason, that worries me: there is a multi-level impact on the memory. 

Consider NUM_NEXTHOPs and allocation of 8 SIDs, in total we will have allocation `in6_addr x 8 x NUM_NEXHTOPs`.
That would be a large memory allocation in next cases:
- staticd
- zebra
- netlink

If you think it is okay to make max-elements to 8, considering the memory impact, please let me know, I will quickly address the subject in the next commit.